setup.py

```diff
@@ -86,6 +86,16 @@ def clean_optional_plugins():
 class Develop(setuptools.command.develop.develop):
     """Custom develop command."""
 
+    user_options = setuptools.command.develop.develop.user_options + [
```

Comment:
Have you tried this?

```python
self.user_options + [....
```

Reply:
That doesn't work, 'self' is not defined there.

Reply:
Sure! I could swear that this was inside a method. ;)